device_backends/pcie/src/PcieBackend.cc

```diff
@@ -72,12 +76,15 @@ namespace ChimeraTK {
       _ioctlPhysicalSlot = LLRFDRV_PHYSICAL_SLOT;
       _ioctlDriverVersion = LLRFDRV_DRIVER_VERSION;
       _ioctlDMA = 0;
-      _readDMAFunction = boost::bind(&PcieBackend::readDMAViaStruct, this, boost::placeholders::_1,
-          boost::placeholders::_2, boost::placeholders::_3, boost::placeholders::_4);
-      _writeFunction = boost::bind(&PcieBackend::writeWithStruct, this, boost::placeholders::_1,
-          boost::placeholders::_2, boost::placeholders::_3, boost::placeholders::_4);
-      _readFunction = boost::bind(&PcieBackend::readWithStruct, this, boost::placeholders::_1, boost::placeholders::_2,
-          boost::placeholders::_3, boost::placeholders::_4);
+      _readDMAFunction = [&](uint8_t bar, uint32_t address, int32_t* data, size_t sizeInBytes) {
```

Comment:
use lambdas instead of bind

Reply:
Ah, you also fixed the double assignment  of the _readFunction :)